test/docker/main.yml

```diff
@@ -25,7 +25,7 @@ RUN apt-get update && apt-get -y install gcc-4.9 g++-4.9 \
 && rm -rf /var/lib/apt/lists/*
 
 # get node
-RUN curl -sL https://deb.nodesource.com/setup_4.x | bash -
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
```

Comment:
version 4 is definitely outdated so it should be updated. i generally try to develop in the version prior to the latest LTS version mostly so it's less likely that any unsupported features will creep into the code and only be discovered during testing potentially forcing rethinking the implementation.

i'm not going to be using this development environment, so this if you like running at the most recent LTS version that works for me.

Reply:
I was under the impression that 11 is the latest version, but it is 'current release' not LTS.
I'll change to it to 8.